doc/ref_model/chapters/chapter08.md

```diff
@@ -98,8 +98,63 @@ The Cloud Services and the Cloud Resources Brokers provide value-added services
 
 <a name="8.2.4"></a>
 ### 8.2.4 Multi-Cloud Interactions Model
-<a name="8.2.4"></a>
-
+<a name="8.2.4.1"></a>
+### 8.2.4.1 Introduction
+To realise a federated cloud requires the definition and agreement on a set of APIs. These APIs should allow each of the parties to interact cooperatively and need to cover the management layer: business management and service operations interactions; as well as the data plane, customer and user, transactions and conversational interfaces.
```

Comment:
These API-s are not yet defined or implemented, therefore Anuket specification projects can only refer these as gaps. 
If Anuket would like to define these API-s I would propose to create a new project for them in Anuket. For the implementation of these API-s in the different cloud solutions there will be a need to direct contributions to Kubernetes and OpenStack.

Reply:
@CsatariGergely  yes agree. The closing section will provide survey of what is available and the applicability of these to support the alternate scenarios.

The focus here is not to specify standards. it is to identify needs and highlight know gap, as per your feedback.

Thank you for feedback.

Reply:
@jbhartley Can you add a Gap in Chapter 10 -- a new Section 10.3.6 API Gaps, and a reference to it as in, " ... only refer these as gaps (see [Section 10.3.6](./chapter10.md#10.3.6))."

@CsatariGergely would that address your concern?

Reply:
@pgoyal01 @CsatariGergely 

I have added Multi-Cloud Gap section into Chapter 10 to cover API Gaps. This is now part of this pull request.

Reply:
@jbhartley Thanks